lib/webauto/web4cucumber.rb

```diff
@@ -74,15 +74,15 @@ def started?
     def browser
       return @browser if @browser && @browser.exists?
       firefox_profile = Selenium::WebDriver::Firefox::Profile.new
-      chrome_caps = Selenium::WebDriver::Remote::Capabilities.chrome()
+      chrome_opts = Selenium::WebDriver::Options.chrome
       safari_caps = Selenium::WebDriver::Remote::Capabilities.safari()
       chrome_switches = []
       if ENV.has_key?("http_proxy") || @http_proxy
         # get rid of the heading "http://" that breaks the profile
         proxy_raw = (@http_proxy || ENV["http_proxy"]).sub(/(\/)+$/, '')
         proxy = proxy_raw.sub(%r{^.+?://}, "")
         proxy_bypass = "localhost,127.0.0.1"
-        firefox_profile.proxy = chrome_caps.proxy = safari_caps.proxy = Selenium::WebDriver::Proxy.new({:http => proxy.sub(%r{^.+?@}, ""), :ssl => proxy.sub(%r{^.+?@}, "")})
+        firefox_profile.proxy = chrome_opts.proxy = safari_caps.proxy = Selenium::WebDriver::Proxy.new({:http => proxy.sub(%r{^.+?@}, ""), :ssl => proxy.sub(%r{^.+?@}, "")})
```

Comment:
it looks like although we set `chrome_opts.proxy` here but it is never used ~ (same to chrome_caps)

Reply:
I haven't do a deep dive into the proxy part, so I just leave it as it is now. Perhaps we can do the cleanup in a follow up PR. WDYT ?